libnetwork/osl/interface_linux.go

```diff
@@ -193,12 +193,12 @@ func (i *nwIface) Statistics() (*types.InterfaceStatistics, error) {
 	}
 
 	return &types.InterfaceStatistics{
-		RxBytes:   uint64(stats.RxBytes),
-		TxBytes:   uint64(stats.TxBytes),
-		RxPackets: uint64(stats.RxPackets),
```

Comment:
These were not by accident to support arm32 ?

Reply:
They are already unit64, there's no conversion.